### PR TITLE
2541 congested mode

### PR DESCRIPTION
**Pull request by gibsonbailey**

### CODE REVIEW

File: codalab/apps/web/tasks.py [hunk 1]
```diff
@@ -816,9 +816,19 @@ def send_mass_email(competition_pk, body=None, subject=None, from_email=None, to
     text = render_to_string("emails/notifications/participation_organizer_direct_email.txt", context)
     html = render_to_string("emails/notifications/participation_organizer_direct_email.html", context)
 
-    mail_tuples = ((subject, text, html, from_email, [e]) for e in to_emails)
+    message_info = []
+    site = Site.objects.get_current()
 
-    _send_mass_html_mail(mail_tuples)
```
Comment (ckcollab) [inline codalab/apps/web/tasks.py]: ah, does this also fix a user context problem?

By the way, we recently had a problem where emailing participants count > 100 seems to not be sending for some reason.. if you setup a free SendGrid account you can send a couple hundred emails to see if you can replicate this recent problem?

File: codalab/apps/web/templates/web/competitions/_submit_results_page.html [hunk 2]
```diff
@@ -25,6 +25,12 @@ <h4>Phase description</h4>
         <p><b>Max submissions total: </b> {{ phase.max_submissions }}</p>
     </div>
 
+    {% if submission_threshold_met %}
+        <div class="alert alert-danger text-center">
+            <h2>We already have too many submissions running. Please come back in a few minutes.</h2>
```
Comment (ckcollab) [inline codalab/apps/web/templates/web/competitions/_submit_results_page.html]: Could you maybe say:

> _Warning: The compute queues are congested, please wait for existing jobs to process before submitting more. Thanks for your patience!_

File: codalab/apps/web/views.py [hunk 3]
```diff
@@ -734,6 +735,17 @@ def get_context_data(self, **kwargs):
             context['last_submission_organization_or_affiliation'] = last_submission.organization_or_affiliation
         except ObjectDoesNotExist:
             pass
+
+        jobs_today = Job.objects.filter(created__year=datetime.today().year,
```
Comment (ckcollab) [inline codalab/apps/web/views.py]: nitpick, should be formatted like this to follow pep8:

```python
        jobs_today = Job.objects.filter(
            created__year=datetime.today().year,
            created__day=datetime.today().day,
            created__month=datetime.today().month
        )
```

If the existing code is formatted wrong it's ok to fix that, all "new code" we should try to style nicely, or if we see something messed up fix it as we go.

File: codalab/apps/web/tasks.py (hunk 1 above)
Comment (ckcollab) [reply codalab/apps/web/tasks.py]: Or: maybe something was branched wrong?

**Comment by ckcollab** [comment]
Also, tests failing atm

File: codalab/apps/web/views.py (hunk 3 above)
Comment (gibsonbailey) [reply codalab/apps/web/views.py]: Sounds great to me.

File: codalab/apps/web/templates/web/competitions/_submit_results_page.html (hunk 2 above)
Comment (gibsonbailey) [reply codalab/apps/web/templates/web/competitions/_submit_results_page.html]: Done! Looks much more professional.

File: codalab/apps/web/tasks.py (hunk 1 above)
Comment (gibsonbailey) [reply codalab/apps/web/tasks.py]: I believe this was a branching problem. I purged my local workspace of old commits, so this problem does not keep appearing.

**Pull request closed without merging**